.github/workflows/release1.yaml

```
@@ -0,0 +1,58 @@
+name: Start release
+
+# There are a lot of builds for different languages that all need to agree on what the version is. To start a release, somebody
+# pushes a branch starting with "release-v". This workflow will then:
+# - Edit all the files that need to know the name of the release.
+# - Commit those files.
+# - Make a new PR for the release.
+# - Add a comment to the PR explaining what's going on.
+# When the release PR is merged, it triggers the second release workflow.
+
+on: create
+
+jobs:
+  kickoff:
+    if: github.event.ref_type == 'branch' && startsWith(github.event.ref, 'release-')
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # If we use the default GITHUB_TOKEN, it won't run our CI job once we create a PR. The DEPLOYMENT_TOKEN is a personal
+        # access token with "repo" permissions.
+        token: ${{ secrets.DEPLOYMENT_TOKEN }}
+    - name: Configure git
+      run: |
+        git config --local user.email ops@ironcorelabs.com
+        git config --local user.name "Leeroy Travis"
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+    - name: Calculate versions from github.ref
+      id: version
+      run: |
+        CURRENT=$(basename ${{ github.ref }})
+        NAME=$(echo ${CURRENT} | sed 's/release-//')
+        NUMBER=$(echo ${NAME} | sed -E -e 's/[^0-9.]+/./g' -e 's/\.+/./g' -e 's/^\.//' -e 's/\.$//')
+        echo "::set-output name=tag::${NAME}"
+        echo "::set-output name=dewey::${NUMBER}"
+    - name: Edit files for release
+      run: .github/set-versions.sh ${{ steps.version.outputs.dewey }}
+    - uses: actions-rs/cargo@v1
+      name: Update Cargo.lock
+      with:
+        command: update
+        args: -p ironoxide-android -p ironoxide-cpp -p ironoxide-java
```

I'm of the opinion that we shouldn't use `actions-rs/cargo` for anything but `cross` support. I would change this section just to 
```suggestion
    - name: Update Cargo.lock
      run: cargo update -p ironoxide-android -p ironoxide-cpp -p ironoxide-java
```
Do you have a different preference?